CONTRIBUTING.rst

```diff
@@ -0,0 +1,345 @@
+<https://github.com/neuropoly/spinalcordtoolbox/issues>`_.
+
+
+Prior to submitting a new issue
+*******************************
+
+Consider the following:
+
+- Please take a few seconds to search the issue database in case the
+  issue has already been raised.
+
+- When reporting an issue, make sure your installation has not been tempered
+  with (and if you can, update to the latest release, maybe the problem was
+  fixed).
+
+
+When submitting an issue
+************************
+
+Consider the following:
+
+- Issue title: try to have a self-descriptive, meaningful issue title,
+  summarizing the problem you see.
+
+  Examples:
+
+  - “installation failure: problem creating launchers”
+  - “sct_crop_image crashes during image cropping”
+  - “add a special mode for squirrel WM/GM segmentation”
+
+
+- Body, in case of an issue:
+
+  - Describe the issue (the title is probably not enough to explain
+    it)
+
+  - Provide steps to reproduce the issue.
+
+    Please try to reproduce your issue using SCT example data as
+    inputs, and to provide a sequence of commands that can reproduce it.
+
+    If this is not possible, try to isolate a minimal input on which the issue
+    happens (eg. one file among a dataset), and to provide this file publicly,
+    or if not possible, privately (coordinate with @jcohenadad).
+
+  - Feel free to add additional information such as screenshots, etc.
+
+
+- Body, in case of a feature request:
+
+  - Try to justify why the feature is desired, and should be in SCT
+
+  - If you have an idea of it, try to provide a *usage scenario*, imagining
+    how the feature would be used (ideally inputs, a sequence of commands,
+    and a desired outcome).
+
+  - If you can, provide references to any theoretical work to help the reader
+    better understand the feature.
+
+Example of bug report here:
+https://github.com/neuropoly/spinalcordtoolbox/issues/444
+
+
+After submitting an issue
+*************************
+
+Consider the following:
+
+- Please try to be of assistance to help developers with additional
+  information, and to confirm resolution of the issue (you may be
+  asked to close the issue to confirm).
+
+
+
+Contributing code
+#################
+
+
+Code contributions can be submitted through github pull requests.
+
+
+Prior to contribute code
+************************
+
+Consider the following:
+
+- Pull requests for bug fixes or new features should be based on the
+  `master` branch.
+
+- The following github documentation may be of use:
+
+  - See `Using Pull Requests
+    <https://help.github.com/articles/using-pull-requests>`_
+    for more information about Pull Requests.
+
+  - See `Fork A Repo <http://help.github.com/forking/>`_ for an
+    introduction to forking a repository.
+
+  - See `Creating branches
+    <https://help.github.com/articles/creating-and-deleting-branches-within-your-repository/>`_
+    for an introduction on branching within GitHub.
+
+- Naming your branch: TODO
```

Comment:
i guess here you can copy/paste your [recent suggestion](https://github.com/neuropoly/spinalcordtoolbox/pull/1916/files)

Reply:
Yes I'll do it once your branch is merged